services/auth/source/ldap/source.go

```diff
@@ -25,7 +25,7 @@ import (
 // Source Basic LDAP authentication service
 type Source struct {
 	Name                  string // canonical name (ie. corporate.ad)
-	Host                  string // LDAP host
+	HostList              string // list containing LDAP host(s)
```

Comment:
So all these hosts should have the same port?

Reply:
@lunny damnn, thats a big miss on my part. 

Can you help with if we should keep the config for each ldap host as below?

`// HostConfig represents the configuration for a single LDAP host
type HostConfig struct {
	Host              string            // Hostname or IP address
	Port              uint16            // Port number
	SecurityProtocol  SecurityProtocol  // Security protocol (LDAP, LDAPS, StartTLS)
	SkipVerify        bool              // Whether to skip TLS certificate verification
	BindDN            string            // Bind DN
	BindPassword      string            // Bind password
	UserBase          string            // User search base
	GroupBase         string            // Group search base
	SearchPageSize    uint32            // Search page size
	Filter            string            // User search filter
	AdminFilter       string            // Admin filter
	RestrictedFilter  string            // Restricted user filter
}

// Source represents the overall LDAP service configuration
type Source struct {
	Name               string        // Canonical name (e.g., corporate.ad)
	Hosts              []HostConfig  // List of host configurations    <-----------------------------
	AttributeUsername  string        // Username attribute
	AttributeName      string        // First name attribute
	AttributeSurname   string        // Surname attribute
	AttributeMail      string        // Email attribute
	AttributeSSHPublicKey string     // SSH Public Key attribute
	AttributeAvatar    string        // Avatar attribute
	AttributesInBind   bool          // Fetch attributes in bind context
	GroupMemberUID     string        // Group attribute containing array of UserUID
	GroupTeamMap       string        // Map LDAP groups to teams
	GroupTeamMapRemoval bool         // Remove user from teams not in corresponding LDAP group
	UserUID            string        // User attribute listed in group
	SkipLocalTwoFA     bool          // Skip 2FA for this source
	authSource         *auth.Source  // Reference to the authSource
}
`

Reply:
Maybe right. But then the UI will become very complex.

Reply:
Agreed, any alternatives you have in mind? 
The UI can have an additional input text box for entering no. of ldap servers first, then providing separate input boxes for each of all common params then say input port value for server 1 and so on..

Reply:
I have no idea. It seems the UI will become much complicated.

Reply:
@silverwind @techknowlogick sorry for the tag, just wanted to know your ideas on this if any.